android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt

```diff
@@ -52,7 +52,7 @@ object Versions {
 
     // see https://android-developers.googleblog.com/2022/06/independent-versioning-of-Jetpack-Compose-libraries.html
     // for Jetpack Compose libraries versioning
-    const val compose_version = "1.5.4"
+    const val compose_bom = "2023.10.01"
```

Comment:
Could we change `AndroidX.compose` to `AndroidX.compose_bom` and remove this?

Reply:
Yes, I did not want to change too many lines, and, either way, after migrating to m-c I still plan to implement version catalogues, so this will soon(-ish) be obsolete.